ldap/servers/slapd/connection.c

```diff
@@ -1482,9 +1482,10 @@ connection_threadmain()
 #endif
 
     while (1) {
-        int is_timedout = 0;
+        int is_timedout;
         time_t curtime = 0;
 
+        is_timedout = 0;
```

Comment:
Why move this assignment?

Reply:
It is a leftover of a previous tentative fix. I was initially thinking that the extra signal_listener were triggered by is_timeout not being reset at each loop. I had a doubt how compiler processed this initial setting and to be sure of that I moved this line

Reply:
Okay, if you undo this bit (ie change back to `int is_timedout = 0;` on a single, line then you have my ack :) Thanks!